src/types/annotator.js

```diff
@@ -14,7 +14,11 @@
  * @prop {string} uri
  * @prop {Target[]} target
  * @prop {string} $tag
- * @prop {boolean} [$highlight]
+ * @prop {boolean} [$highlight] -
+ *   Flag indicating that this annotation was created using the "Highlight" button,
+ *   as opposed to "Annotate".
+ * @prop {boolean} [$orphan] -
+ *   Flag indicating whether this annotation was successfully anchored in the document
```

Comment:
I think this has it backwards: it makes it sound like this flag is _set_ when an annotation is successfully anchored.